commands/client.go

```diff
@@ -126,16 +126,17 @@ func GetClientSearchCmd() (cli.Command, error) {
 	return NewCommand(CommandArgs{
 		Path:         []string{cst.NounClient, cst.Search},
 		SynopsisText: fmt.Sprintf("%s (<role> | --role)", cst.Search),
-		HelpText: fmt.Sprintf(`Search for %[1]ss attached to a given %[5]s in %[2]s
+		HelpText: `Search for clients attached to a given role in DevOps Secrets Vault
 
 Usage:
-   • %[1]s %[4]s %[3]s
-   • %[1]s %[4]s --role %[3]s
-`, cst.NounClient, cst.ProductName, cst.ExampleRoleName, cst.Search, cst.NounRole),
+   • role search gcp-svc-1
+   • role search --role gcp-svc-1 --sort asc
```

Comment:
The prior doc examples use tokens and formatting. This uses a hard coded example. Was this intentional?

Reply:
I think it is because you can easily search in the code by copying this text